python/ql/src/Expressions/Formatting/AdvancedFormatting.qll

```diff
@@ -61,10 +63,12 @@ predicate brace_sequence(PossibleAdvancedFormatString fmt, int index, int len) {
     )
 }
 
+/** Holds if index `index` in the format string `fmt` contains an escaped `{`. */
 predicate escaped_brace(PossibleAdvancedFormatString fmt, int index) {
     exists(int len | brace_sequence(fmt, index, len) | len % 2 = 0)
 }
 
+/** Holds if index `index` in the format string `fmt` contains a left curly brace that acts as an escape. */
```

Comment:
It looks as if US English uses just brace (https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/special-characters).

```suggestion
/** Holds if index `index` in the format string `fmt` contains a left brace `{` that acts as an escape character. */
```